src/syntax-tree/ast.ts

```diff
@@ -1595,8 +1595,12 @@ export class MethodCallModifier extends Modifier {
             areEqualTypes(providedContext?.expressionToLeft?.returns, type)
         );
 
-        //#260
-        if (this.returns === DataType.Void && providedContext?.lineStatement instanceof VarOperationStmt) {
+        //#260/#341
+        if (
+            this.returns === DataType.Void &&
```

Comment:
The place where we usually do these type checks does not have access to everything it needs. It is much easier to just do it here.